Jenkinsfile

```diff
@@ -0,0 +1,16 @@
+@Library('linkurious-shared')_
+
+nodeJob {
+  // General
+  projectName = "node-activedirectory"
```

Comment:
same remark as for apidoc, I think actually just adding a @linkurious namespace in the package.json could solve the collision risk

Reply:
haha perfect, that's what I suggested on apidoc without reading this. Will do it on monday and merge.

Reply:
done